lib/utils/injectRefreshEntry.js

```diff
@@ -37,12 +37,11 @@ function injectRefreshEntry(originalEntry, options, compiler) {
   let resourceQuery = {};
 
   if (compiler && compiler.options && compiler.options.devServer) {
```

Comment:
I’m not sure if we want to handle this case, but `compiler` be an array (check out https://webpack.js.org/api/node/#multicompiler)

Reply:
In that case, we’d have to `forEach` `compiler.compilers`

Reply:
from testing, it appears that webpack executes each one independently, so that the compiler variable is still just an object. in other words, the plugin executes _n_ number of times for _n_ number of entries.

I guess that's also somewhat documented here in that link you sent me:

>The MultiCompiler module allows webpack to run multiple configurations in **separate** compilers. If the options parameter in the webpack's NodeJS api is an array of options, webpack applies **separate** compilers 

so I think we're ok here?

Reply:
(also `compiler.compilers` is undefined when I look at it)